ruby/red-arrow/README.md

```diff
@@ -50,3 +50,20 @@ table = Arrow::Table.load("/dev/shm/data.arrow")
 # Process data in table
 table.save("/dev/shm/data-processed.arrow")
 ```
+
+## development
+```
+$ cd ruby/red-arrow
+$ bundle install
+$ bundle exec rake test
+```
+
+### Mac OS
+```
+$ cd ruby/red-arrow
+$ bundle install
+$ brew install apache-arrow --head
+$ brew install apache-arrow-glib --head
+$ export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
```

Comment:
Can we remove this with the following patch?

```diff
diff --git a/ruby/red-arrow/ext/arrow/extconf.rb b/ruby/red-arrow/ext/arrow/extconf.rb
index 5ba9f4c2e..9e92bd316 100644
--- a/ruby/red-arrow/ext/arrow/extconf.rb
+++ b/ruby/red-arrow/ext/arrow/extconf.rb
@@ -25,6 +25,19 @@ if arrow_pkg_config_path
   ENV["PKG_CONFIG_PATH"] = pkg_config_paths.join(File::PATH_SEPARATOR)
 end
 
+checking_for(checking_message("Homebrew")) do
+  platform = NativePackageInstaller::Platform.detect
+  if platform.is_a?(NativePackageInstaller::Platform::Homebrew)
+    openssl_prefix = `brew --prefix openssl@1.1`.chomp
+    unless openssl_prefix.empty?
+      PKGConfig.add_path("#{openssl_prefix}/lib/pkgconfig")
+    end
+    true
+  else
+    false
+  end
+end
+
 unless required_pkg_config_package([
                                      "arrow",
                                      Arrow::Version::MAJOR,
```

Reply:
BTW, could you show your `$(brew --prefix)/lib/pkgconfig/arrow*.pc`?

Reply:
@kou
Thank you for reviewing! 

I added the following patch and removed the comment.
https://github.com/apache/arrow/pull/11188
This change worked correctly in my environment!

> BTW, could you show your $(brew --prefix)/lib/pkgconfig/arrow*.pc?

```
> ls $(brew --prefix)/lib/pkgconfig/arrow*.pc

/opt/homebrew/lib/pkgconfig/arrow-compute.pc            /opt/homebrew/lib/pkgconfig/arrow-glib.pc
/opt/homebrew/lib/pkgconfig/arrow-csv.pc                /opt/homebrew/lib/pkgconfig/arrow-json.pc
/opt/homebrew/lib/pkgconfig/arrow-dataset-glib.pc       /opt/homebrew/lib/pkgconfig/arrow-orc-glib.pc
/opt/homebrew/lib/pkgconfig/arrow-dataset.pc            /opt/homebrew/lib/pkgconfig/arrow-orc.pc
/opt/homebrew/lib/pkgconfig/arrow-filesystem.pc         /opt/homebrew/lib/pkgconfig/arrow-python-flight.pc
/opt/homebrew/lib/pkgconfig/arrow-flight-glib.pc        /opt/homebrew/lib/pkgconfig/arrow-python.pc
/opt/homebrew/lib/pkgconfig/arrow-flight.pc             /opt/homebrew/lib/pkgconfig/arrow.pc
```

Reply:
Thanks.

Could you also show test contents of them?